docs/CM_Replication.adoc

```diff
@@ -166,6 +166,49 @@ try (ReplicatedMap clusterOnHost1 = createCluster(CLUSTER_YAML, 1);
 
 This example is available in the Chronicle Map Enterprise Demo - for access contact info@chronicle.software
 
+=== Implementing Custom Serializers
+
+In Chronicle Map, custom serializers can be applied to `ChronicleMapBuilder` using `valueMarshaller()` method.
+For example:
+
+[source, java]
+----
+ChronicleMapBuilder<Integer, MyDto> builder = ChronicleMapBuilder.of(Integer.class,
+                MyDto.class)
+                .entries(10)
```

Comment:
Indentation should be more natural (different levels for parameter and subsequent method calls) - perhaps use IDEA to reformat?